addons/skin.estouchy/xml/ViewsThumbnail.xml

```diff
@@ -43,7 +43,7 @@
 					<height>86</height>
 					<texture background="true">$INFO[Listitem.Icon]</texture>
 					<aspectratio>keep</aspectratio>
-					<visible>[Container.Content() + !Window.IsVisible(Pictures)] | Container.Content(Files) | Container.Content(Genres) | Container.Content(Years) | Container.Content(Directors) | Container.Content(Studios) | Container.Content(Countries) | Container.Content(Tags) | Container.Content(Playlists) | [Window.IsVisible(Pictures) + String.IsEmpty(Container.FolderPath)]</visible>
+					<visible>[Container.Content() + !Window.IsVisible(Pictures)] | Container.Content(Files) | Container.Content(Games) | Container.Content(Genres) | Container.Content(Years) | Container.Content(Directors) | Container.Content(Studios) | Container.Content(Countries) | Container.Content(Tags) | Container.Content(Playlists) | [Window.IsVisible(Pictures) + String.IsEmpty(Container.FolderPath)]</visible>
```

Comment:
The special handling of empty content type for pictures window shouldnt be needed anymore since we also have container.content(images) since v16 or v17, not sure.
just "Container.Content()  | Container.Content(Files) | Container.Content(Games) | Container.Content(Genres) | Container.Content(Years) | Container.Content(Directors) | Container.Content(Studios) | Container.Content(Countries) | Container.Content(Tags) | Container.Content(Playlists)" should be fine i think.